modules/crm/views/js-templates/new-customer.php

```diff
@@ -1,6 +1,10 @@
 <div class="erp-customer-form">
 
-    <?php do_action( 'erp-crm-customer-form-top' ); ?>
+    <# if ( 'company' == data.type ) { #>
+        <?php do_action( 'erp_crm_company_form_top' ); ?>
```

Comment:
This is a wrong logic, you are using `if` in javascript and `do_action` in PHP. In this case these two will always run. So keeping it simple, remove the company one keep contact one only. This can serve both.